tests/numpy/test_misc.py

```diff
@@ -94,6 +94,15 @@ def test_reshape(nps_app_inst):
     assert nps.allclose(ba.reshape(shape=(6, 4), block_shape=(6, 4)),
                         nps.reshape(ba, shape=(6, 4)))
 
+def test_all(nps_app_inst):
```

Comment:
Can you add a test that evaluates bools and floats directly? Something simple is fine -- just make sure you cover both cases, where `all` should evaluate to true and false.

Also, while the test for int is fine (you can do bool the same way), for the float test, test the output against what numpy does. I'm not sure whether it evaluates a float `0.0` as true or false.

Finally, please add an assertion for the dtype of all tests (e.g. `assert nps.all(b).dtype is np.bool`).

Reply:
Added all suggestion.s

Reply:
Almost there. Could you add type checks for the bool and int types as you have with int (e.g. `assert nps.all(true_int).dtype is np.bool`)?

Reply:
Done.